tests/test_oi.py

```diff
@@ -10,7 +10,7 @@ def test_oi_1(self):
         '''OI = (ADD+MUL)/[(LOAD+STORE)*word_size]; word_size=8(double),4(float)
         Equation = v1[i2][i1] = 3*v2[i2][i1] + 2*v3[i2][i1];
         '''
-        load = 3.0
+        load = 2.0
```

Comment:
Since we have multiple ways of counting loads, should the OI tests be extended to cover them all?